content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -1194,7 +1200,7 @@ strings in the general form:
   request event, encoded as:
   - two bytes in network byte order (big-endian) indicating the length in
     bytes of the ID as a UTF-8 string
-  - the ID as a UTF-8 string
+  - the ID encoded as a UTF-8 string (i.e. one UTF-8 byte per character)
```

Comment:
I don't know what "one UTF-8 byte per character" means. Isn't that a contradiction?

Assuming there are non-ascii characters in the ID, then those characters will require more than one byte to encode.

Reply:
I don't recall what I was thinking... this looks better to me:

```suggestion
  - the ID encoded as a UTF-8 string
```